damus/Models/ContentFilters.swift

```diff
@@ -25,7 +25,7 @@ enum FilterState : Int {
 
 /// Simple filter to determine whether to show posts with #nsfw tags
 func nsfw_tag_filter(ev: NostrEvent) -> Bool {
-    return ev.referenced_hashtags.first(where: { t in t.hashtag == "nsfw" }) == nil
+    return ev.referenced_hashtags.first(where: { t in t.hashtag == "nsfw" }) == nil && References<ContentWarningTag>(tags: ev.tags).first == nil
```

Comment:
dont we need to check the reason == nsfw in the content warning case?

Reply:
We should consider any post with the "content-warning" tag as sensitive.
Therefore, there is no need to check the "reason."

Reply:
This is a post without reason.
https://nostter.app/nevent1qqswynxvz0wth52wjq3xyq7l795q3w0fe83ym0ewetnx8u9gslrhmhswwl5pw
```JSON
{
  "content": "に",
  "created_at": 1718237816,
  "id": "e24ccc13dcbbd14e90226203dff16808b9e9c9e24dbf2ecae663f0a887c77dde",
  "kind": 1,
  "pubkey": "1a35b54ef7752af54cacbeedf0f349e320f0a2ee50142883134c3ee31879ce71",
  "sig": "3a1dc4873e2a8527ecfce2526af8701702e0d1bb3b4c4ecf45e8026cae27ceb84f40144e8409ca4a325e8e93f39fcba75ddf1c7ad0e57ba53f7e538a4da92b2a",
  "tags": [
    [
      "content-warning"
    ]
  ]
}
```